pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -203,6 +203,8 @@ rec {
             ++ optional (elem "host"   configurePlatforms) "--host=${stdenv.hostPlatform.config}"
             ++ optional (elem "target" configurePlatforms) "--target=${stdenv.targetPlatform.config}";
 
+          CLICOLOR_FORCE = 1;
+
```

Comment:
stdenv people: Please let me know if there's a better place to put it than here. It doesn't feel "right" here

Reply:
Put it in stdenv/setup.sh?

Reply:
You could probably put it in impureEnvVars. At least, there are definitely times when you may not want colors, for instance in Eshell.